pkg/rules/slp066.go

```
@@ -0,0 +1,84 @@
+// precisely matching mutex guards to specific map variables requires full
+// AST analysis which is out of scope for diff-based linting.
+type SLP066 struct{}
+
+func (SLP066) ID() string                { return "SLP066" }
+func (SLP066) DefaultSeverity() Severity { return SeverityBlock }
+func (SLP066) Description() string {
+	return "concurrent map access without mutex protection"
+}
+
+// mapIndexPattern detects map read/write operations like m[key] or m[key] = value.
+var mapIndexPattern = []string{"[", "]"}
```

golangci-lint (2.11.4)

[error] var mapIndexPattern is unused

(unused)